src/app/organization/organization-routing.module.ts

```diff
@@ -121,11 +125,34 @@ const routes: Routes = [
         },
         {
           path: 'sms-campaigns',
-          component: SmsCampaignsComponent,
           data: { title: extract('SMS Campaigns'), breadcrumb: 'SMS Campaigns' },
-          resolve: {
-            smsCampaigns: SmsCampaignsResolver
-          }
+          children: [
+            {
+              path: '',
+              component: SmsCampaignsComponent,
+              resolve: {
+                smsCampaigns: SmsCampaignsResolver
+              }
+            },
+            {
+              path: ':id',
+              component: ViewSmsCampaignComponent,
+              data: { title: extract('View SMS Campaign'), routeResolveBreadcrumb: ['smsCampaign', 'campaignName'] },
+              resolve: {
+                smsCampaign: SmsCampaignResolver
+              },
+              children : [
+                {
+                  path: 'campaign',
+                  component: CampaignTabComponent,
+                  data: { title: extract('Campaign'), routeParamBreadcrumb: false },
+                  resolve: {
+                  smsCampaign: CampaignResolver
+                }
```

Comment:
Ensure proper indentation.
```suggestion
                  }
```